Jenkinsfile

```diff
@@ -12,9 +12,11 @@
 def runTests = true
 
 // Only keep the 10 most recent builds.
-properties([[$class: 'jenkins.model.BuildDiscarderProperty', strategy: [$class: 'LogRotator',
-                                                                        numToKeepStr: '50',
-                                                                        artifactNumToKeepStr: '20']]])
+// NOTE: With Pipeline 1.14 and later combined with Jenkins 1.642 and later, this may need to be changed to
```

Comment:
> with 1.625.3 and Pipeline 1.13, `jenkins.model.BuildDiscarderProperty` does not work

Because in that world, this class does not exist. That is why you need to specify the older class name, which does exist.